src/mscorlib/src/System/Globalization/TextInfo.cs

```diff
@@ -684,6 +693,40 @@ private static bool IsLetterCategory(UnicodeCategory uc)
                  || uc == UnicodeCategory.OtherLetter);
         }
 
+
+        /// 
+        ///  Checks if the current culture is Dutch (i.e. "nl-BE" or "nl-NL") to
+        ///  determine if special casing rules should be applied. (Dutch Titlecasing)
+        ///
+        private bool IsDutchCulture(String str)
+        {
+            // Validate inputs
+            if (str == null)
+            {
+                throw new ArgumentNullException("string");
+            }
+
+            // Ignore if not Dutch
+            return IndexOfStringOrdinalIgnoreCase("nl-BE,nl-NL", _cultureName, 0, 11) != -1;
```

Comment:
>return IndexOfStringOrdinalIgnoreCase("nl-BE,nl-NL", _cultureName, 0, 11) != -1; [](start = 12, length = 80)

this need to do something CultureName.StartsWith("nl-", StringComparison.OrdinalIgnoreCase)

by the way, does nl-BE & nl-NL are the only Dutch cultures? do consider de-DE too? how did you get the list?